specification/databox/resource-manager/Microsoft.DataBox/preview/2018-01-01/databox.json

```diff
@@ -536,103 +555,19 @@
           }
         },
         "x-ms-examples": {
-          "JobsCancel": {
-            "$ref": "./examples/JobsCancel.json"
-          }
-        }
-      }
-    },
-    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DataBox/jobs/{jobName}/copyLogsUri": {
```

Comment:
What is the migration path for deprecation of this API? It appears to be in active current use.

Reply:
Till now are only active user is our own portal and we have stopped using these apis, our production systems have already stopped supporting these and see no usage/failure because of its remove. Hence as we go GA on 24th september we dont want the apis to be available for our GA sdk

Reply:
Here is the plan
1) For Apis GetCopyLogsUri, DownloadShippingLabelUri, ReportIssue have zero users and the api has been disabled and non exisitent in our service for more than a month, so we are removing it from sdk also
2) ListSecrets is continued to be supported from service but is no longer support in the new generated client dll, anyone calling the api will be returned a proper response

Just FYI we have only testing (internal) sdk users, post we go GA we will have more sdk users

Reply:
@sojain There are no external users of this API? Why not take this to a new API version? These are significant changes in an API version. Are you planning to move to a stable API for your GA?

Reply:
@KrisBash , yes this pr is to show the diffs, as soon as I have a sign off ill update this spec to stable and only post its completion we will generate a new dll for customer

Reply:
@sojain Thanks